### ISSUE TITLE
TypeError: assert.isAtLeast is not a function

### ISSUE THREAD

**Issue by damianfrizzi** [theintern/intern#588]
Intern version: 3.0.6

I'm experiencing a TypeError when trying to make an assertion according to [chai docs](http://chaijs.com/api/assert/#isatleastvaluetocheck-valuetobeatleast-message). It happens for the methods isAtLeast and isAtMost while e.g. isBelow is working properly.
I've tried upgrading the chai version from currently 3.0.0 to 3.5.0. Unfortunately that didn't change anything.

I broke down the test suite to this failing test:

```
define(function (require) {
var registerSuite = require('intern!object');
var assert = require('intern/chai!assert');

registerSuite({
name: 'Test',
test: function() {
assert.isAtLeast(5, 2, '5 is greater or equal to 2');
}
});
});
```

**Comment by jason0x43** [comment]
The problem is due to the old chai dependency. Upgrading chai in an Intern install is a bit tricky with node's de-duping and Intern's symlinking, but when you do, your test should pass (it does for me, at least). The next release should include an updated chai.

**Comment by sholladay** [comment]
As a workaround, you can do `npm install --save chai` and then simply import chai a bit differently in your tests.

````javascript
define(
    [
        'intern/dojo/node!chai'
    ],
    function (chai) {
        const assert = chai.assert;
    }
);
````

I am not aware of any problems with this approach. I have had to do so myself.

**Comment by laurentpayot** [comment]
Same bug here.

**Comment by kitsonk** [comment]
Please note, this is not a bug.  The documentation you are looking at is for a newer version of Chai's assertion library than what is available with Intern 3.0.6 and a workaround for updating the dependency is listed above.

**Comment by jason0x43** [comment]
Closing since this issue is specific to Intern 3
